libtransact/src/state/merkle/change_log.rs

```diff
@@ -19,7 +19,7 @@ use crate::protos::merkle;
 use crate::protos::{FromNative, FromProto, IntoNative, IntoProto, ProtoConversionError};
 use protobuf::Message;
 
-use super::merkle::StateDatabaseError;
+use super::error::StateDatabaseError;
```

Comment:
This was probably intended for the previous commit.

Reply:
Not, really - moving this file, the super of this is now `state::merkle`, where as prior to this commit, it was `state` and this module was a peer of `merkle`